test/integration/client-side-encryption/client_side_encryption.prose.test.js

```diff
@@ -1106,6 +1110,75 @@ describe('Client Side Encryption Prose Tests', metadata, function () {
 
     it.skip('Via bypassAutoEncryption', () => {}).skipReason =
       'TODO(NODE-2422): Implement "Bypass spawning mongocryptd" tests';
+
+    describe('via loading shared library', function () {
+      let clientEncrypted;
```

Comment:
We're missing logic to skip this test when the shared library is not present: 

>   IMPORTANT: This test requires the crypt_shared_ library be loaded. If the crypt_shared_ library is
   not available, skip the test.

I think we can use the `CRYPT_SHARED_LIB_PATH` environment variable to determine whether or not we're testing with the shared library.